### PR TITLE
Acceptance test fixup

### PR DESCRIPTION
**Pull request by alfrunes**
- [X] Replace minio with boto3 (S3) library
- [x] Use pytest semantics for loading coftest parameters into test fixtures
  - Gave up on using fixtures since the setup is too convoluted and relying on just-in-time loading of options. 
- [x] Remove hard-coded values for mongo URL and docker compose labels in `CliClient`

### CODE REVIEW

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline --fast

File: tests/run.sh [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 DIR=$(readlink -f $(dirname $0))
 
+pip install boto3 # FIXME
+
```
Comment (tranchitella) [inline tests/run.sh:21]: This is not great; however, I understand it is required to make it work.

**Review by tranchitella — APPROVED** [review]
I approve; however, the `boto3 installation` as part of `tests/run.sh` should not be there and I highly suggest updating the upstream docker image instead.

File: tests/run.sh (hunk 1 above)
Comment (alfrunes) [reply tests/run.sh:21]: Sorry, I forgot to put link in the PR, but I created one here: https://github.com/mendersoftware/mender-test-containers/pull/569
I will remove the top commit before I merge (once the mender-test-container is published).

**Pull request merged**